api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceImplTest.java

```diff
@@ -53,23 +53,26 @@
 
     private Extension ext5;
 
-
     @Before
     public void setUp() throws Exception {
 
+    	// clear up from previous tests
```

Comment:
The current tests use DirtiesContext and the semantics have changed in the spring/hibernate versions used in platform 2.0. I had to remove the DirtiesContext annotation to get the tests to run and had to make changes in the appframework tests to compensate.

After DirtiesContext is invoked, it is impossible to login into the platform hence tests fail.